### PR TITLE
[SYM-3916] Tailscale Access Strategy Example

### PR DESCRIPTION
**Pull request by Arianna2028**
Implements an example for the new Tailscale Access Strategy.

### CODE REVIEW

**Comment by Arianna2028** [comment]
Current dependencies on/for this PR:
* main
  * **PR #11** <a href="https://app.graphite.dev/github/pr/symopsio/quickstarts/11" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/symopsio/quickstarts/11?utm_source=stack-comment).

File: tailscale_ssh_access/main.tf [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+      {
+        name           = "duration"
+        type           = "duration"
+        allowed_values = ["30m", "1h"]
+        required       = true
+      }
+    ])
+  }
+}
+
```
Comment (llam15) [inline tailscale_ssh_access/main.tf:156]: In my more recent examples, I've been sectioning this off with
```suggestion
############ Basic Environment Setup ##############
```

**Review by llam15 — COMMENTED** [review]
just a couple comments. We should also add a row to the top-level ReadMe table of contents.

Also if possible, are you able to get a screen recording of the flow?

File: tailscale_ssh_access/main.tf [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  integrations = {
+    slack_id = sym_integration.slack.id
+  }
+}
+
+resource "sym_integration" "slack" {
+  type = "slack"
+  name = "main-slack"
+
+  external_id = "T123ABC"
```
Comment (druiz) [inline tailscale_ssh_access/main.tf:173]: Worth commenting out that this is supposed to be the Slack workspace ID?

**Review by druiz — APPROVED** [review]

**Review by llam15 — APPROVED** [review]

**Pull request merged**